src/zopfli/zopfli_bin.c

```diff
@@ -99,10 +108,17 @@ static void CompressFile(const ZopfliOptions* options,
     SaveFile(outfilename, out, outsize);
   } else {
     size_t i;
+/* Windows workaround for stdout output. */
+#if _WIN32
```

Comment:
Is it possible to avoid this platform specific mode in this source code?

Does windows have maybe some other way to externally change the mode of an executable to binary?

Otherwise, it's always possible to write to a file.